metabase/metabase
# Move pinning items in collection to menu

Epic https://github.com/metabase/metabase/issues/23222

How to test:
- Open a collection with unpinned items
- Make sure it is possible to pin the item with the menu option

<img width="1324" alt="Screenshot 2022-06-17 at 16 13 32" src="https://user-images.githubusercontent.com/8542534/174306015-27f62ed7-02e1-4664-9e6a-87f8dcfce52b.png">
